testing/mocks/generic.go

```diff
@@ -38,6 +38,17 @@ import (
 	"github.com/optakt/flow-dps/rosetta/object"
 )
 
+// Offsets used to ensure different flow identifiers.
+const (
+	offsetBlock       = 2 << 1
+	offsetSeal        = 2 << 2
+	offsetCollection  = 2 << 3
+	offsetGuarantee   = 2 << 4
+	offsetTransaction = 2 << 5
+	offsetResult      = 2 << 6
+	offsetEvent       = 2 << 7
+)
```

Comment:
Yeah, OK, why not.

Reply:
In retrospect, I'm not sure what I was hoping to do by doing the offsets like this. It doesn't prevent IDs from overlapping, only makes it very unlikely for the last offset kinds.

Since the offset for blocks is 2 and the offset for seals is 4, seal ID number 1 will be the same as block ID number 5 (or something like that).

Maybe my idea wasn't all wrong but instead of having shifts by powers of two, the offsets could be smth like  16/32/48/64/... to guarantee a non-overlapping range of 16 per type of resource, since AFAIR we never generate more than 6-8 of each resource at the moment.

WDYT?